netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -104,6 +95,50 @@ public Response act(final Request req) throws IOException {
     }
 
     /**
+     * Invite friend into the bout.
+     * @param bout Bout
+     * @param msg Message
+     * @param href Href
+     * @throws IOException If there is some problem inside
+     */
+    private void invite(final Bout bout, final StringBuilder msg,
+        final Href href) throws IOException {
+        final Iterator<String> invite = href.param("invite").iterator();
+        while (invite.hasNext()) {
+            final String friend = invite.next();
+            try {
+                bout.friends().invite(friend);
+            } catch (final Friends.UnknownAliasException ex) {
+                this.discard(bout);
+                throw new RsFailure(ex);
+            }
+            msg.append(String.format(", \"%s\" invited", friend));
```

Comment:
@dmzaytsev the text is fine. I was refering to the location. Perhaps move it to line 111?